### ISSUE TITLE
"Error Cannot read property 'tree' of undefined" on click event

### ISSUE THREAD

**Issue by mletov** [ludo/jquery-treetable#187]
Hi, Ludo!
Help me, please. 
I need load new branch by ajax. I tried your example:

```
 $(".SPaddBranch").on("click", addBranch);
    function addBranch()
    {
        var node = $(".tbl-targets").treetable("node", "trg-146");
        ....
    }

```
but i got "Error Cannot read property 'tree' of undefined" 

If i simply write
`var node = $(".tbl-targets").treetable("node", "trg-146");`
not in event handler - all good.
I read similar error https://github.com/ludo/jquery-treetable/labels/support
but it is different case.
